source/NVDAObjects/__init__.py

```diff
@@ -818,6 +818,15 @@ def reportFocus(self):
 		"""
 		speech.speakObject(self,reason=controlTypes.REASON_FOCUS)
 
+	def _get_isSelectionAnchoredAtStart(self):
+		"""Indicates if the selection is anchored at the start.
+		The anchored position is the end that doesn't move when extending or shrinking the selection.
+		If the selection is anchored at the end or there is no information this is C{False}.
+		@return: C{True} if the selection is anchored at the start else C{False}
+		@rtype: bool
+		"""

```

Comment:
It strikes me as odd that we create this private variable here which gets set in a completely different class. That is, when you're reading this code, it's not clear where this gets set. I'm thinking the base implementation should just be a class variable with a default value and then the classes which implement this can either implement their own property or override with a simple instance variable.